functions/Backup-DbaDatabaseCertificate.ps1

```diff
@@ -179,24 +202,24 @@ Exports all certificates named CertTDE on the specified SQL Server, not specifyi
 						ComputerName   = $server.NetName
 						InstanceName   = $server.ServiceName
 						SqlInstance    = $server.DomainInstanceName
-						Database       = $db.name
-						Certificate    = $certname
-						ExportPathCert = $exportpathcert
-						ExportPathKey  = $exportpathkey
+						Database       = $db.Name
+						Certificate    = $certName
+						exportPathCert = $exportPathCert
+						exportPathKey  = $exportPathKey
 						Status         = "Failure: $exception"
 					}
-					Stop-Function -Message "$certname from $db on $instance cannot be exported." -Continue -Target $cert -InnerErrorRecord $_
+					Stop-Function -Message "$certName from $db on $instance cannot be exported." -Continue -Target $cert -InnerErrorRecord $_
 				}
 			}
 		}
 	}
 	
 	process {
 		if (Test-FunctionInterrupt) { return }
-		foreach ($instance in $sqlinstance) {
+		foreach ($instance in $sqlInstance) {
```

Comment:
This one should actually be `$SqlInstance` because it is a parameter for the function.